rally_ovs/plugins/ovs/scenarios/ovn_nb.py

```diff
@@ -226,19 +226,38 @@ def create_routed_lport(self, lport_create_args = None,
                             port_bind_args = None,
                             create_acls = True,
                             name_space_size = 1,
-                            network_policy_size = 1):
+                            network_policy_size = 1,
+                            ext_cmd_args = {}):
         lswitches = self.context["ovn-nb"]
         ip_offset = lport_create_args.get("ip_offset", 1) if lport_create_args else 1
 
         iteration = self.context["iteration"]
         lswitch = lswitches[iteration % len(lswitches)]
         ip_start_index = iteration / len(lswitches) + ip_offset
 
+        start_cmd = ext_cmd_args.get("start_cmd", None)
+        if start_cmd and iteration == start_cmd.get("iter", -1):
+               self.handle_cmd(start_cmd)
+        stop_cmd = ext_cmd_args.get("stop_cmd", None)
+        if stop_cmd and iteration == stop_cmd.get("iter", -1):
+               self.handle_cmd(stop_cmd)
+
         self.configure_routed_lport(lswitch, lport_create_args,
                                     port_bind_args, ip_start_index,
                                     name_space_size, network_policy_size,
                                     create_acls)
 
+    @scenario.configure()
+    def handle_cmd(self, cmd_args = {}):
+        sandbox_size = cmd_args.get("size", 0)
```

Comment:
Maybe we can find a better name than "size", what do you think? By looking at the json file it's not so clear that it refers to the number of sandboxes to run the command on.

Reply:
@dceara ack, I have not too imagination actually. What about `num_sanboxes` ?